src/Components/DesignAdjustments.tsx

```diff
@@ -0,0 +1,27 @@
+import { Obj, connect } from 'scrivito'
+import { isHomepage } from '../Objs/Homepage/HomepageObjClass'
+import { Helmet } from 'react-helmet-async'
+
+export const DesignAdjustments = connect(function DesignAdjustments() {
+  const root = Obj.root()
+  if (!isHomepage(root)) return null
+
+  const styles: string[] = []
+
+  const dropShadow = root.get('siteDropShadow')
+  styles.push(
+    `--jr-card-box-shadow: ${
+      dropShadow ? '0 0.375rem 1.5rem 0 rgba(140, 152, 164, 0.25)' : 'none'
+    };`,
+  )
+
+  const roundedCorners = root.get('siteRoundedCorners')
+  styles.push(`--jr-card-border-radius: ${roundedCorners ? '0.5rem' : '0'};`)
```

Comment:
Having too specific CSS rules in the JS is a smell. As long as we only deal with on/off switches, how about this:
With the [fallbacks in the CSS](https://github.com/Scrivito/scrivito-portal-app/pull/460/files#diff-ac1966f8adc171be5f6109146d471ed6872e1065794f6b96e6569dbd3d6a4b20R75-R76) we could simplify this to do one of two things:
1. Don't set the variable. This will effectively activate the default from the CSS.
2. Set it to a simple null value that works independently of the actual CSS. E.g, `unset`, `none`, `0px`. This will override and effectively disable the default from the CSS.

* [ ] FYI maybe rethink